olive/model/handler/pytorch.py

```diff
@@ -152,6 +153,31 @@ def load_model(self, rank: int = None) -> torch.nn.Module:
 
         return model
 
+    def get_component_model(self, component: HfComponent, rank: Optional[int] = None) -> "PyTorchModelHandler":
+        if component.component_func is None:
+            logger.debug("component_func is not provided, using hf_config to get component")
+            model_component = self.load_hf_model(self.model_path)
+        else:
+            user_module_loader = UserModuleLoader(self.model_script, self.script_dir)
+            model_component = user_module_loader.call_object(component.component_func, self)
+
+        # the second default parameter is to fix ruff b023:
+        # https://docs.astral.sh/ruff/rules/function-uses-loop-variable/
```

Comment:
the functionality is already in https://github.com/microsoft/Olive/blob/main/olive/model/handler/mixin/hf_config.py#L62